vms/platformvm/state/diff.go

```diff
@@ -88,6 +88,7 @@ func NewDiff(
 		parentID:      parentID,
 		stateVersions: stateVersions,
 		timestamp:     parentState.GetTimestamp(),
+		caminoDiff:    *newCaminoDiff(),
```

Comment:
Same here, if pointer does not brings new issues, change to pointer

Reply:
oh, thanks, I forgot to update that